python/paddle/fluid/layers/control_flow.py

```diff
@@ -2284,6 +2306,8 @@ def append_conditional_block_grad(self, parent_block, inside_block,
 def copy_var_to_parent_block(var, layer_helper):
     if var is None:
         return None
+    if isinstance(var, (bool, float, six.integer_types)):
```

Comment:
```suggestion
    if  not isinstance(var, Variable):
```

只要不是variable类型都直接返回。